go_modules/lib/dependabot/go_modules/file_parser.rb

```diff
@@ -65,8 +56,8 @@ def dependency_from_details(details)
         )
       end
 
-      def module_info
-        @module_info ||=
+      def required_packages
+        @required_packages ||=
           SharedHelpers.in_a_temporary_directory do |path|
             SharedHelpers.with_git_configured(credentials: credentials) do
```

Comment:
Idea: iff we don't `go list -m` any more, do we still need git configured here? I'd expect parsing the file and converting to json to be local-only...

Reply:
Yes, you're absolutely right! 👍 will get rid of this